pinot-core/src/main/java/org/apache/pinot/core/geospatial/transform/function/ScalarFunctions.java

```diff
@@ -98,4 +99,15 @@ public static String stAsText(byte[] bytes) {
     GeometryUtils.setGeometry(geometry);
     return GeometrySerializer.serialize(geometry);
   }
+
+  /**
+   * Gets the H3 hexagon address from the location
+   * @param latitude latitude of the location
+   * @param longitude longitude of the location
+   * @param resolution H3 index resolution
+   * @return the H3 index address
+   */
+  public static long getHexagonAddr(double latitude, double longitude, int resolution) {
```

Comment:
Annotate with `ScalarFunction`

Reply:
We should also add the scalar function with 2 arguments, where the first one is `byte[]` of serialized point

Reply:
We should put longitude before latitude to be consistent with other geo functions